dot/parachain/types/bitvec.go

```diff
@@ -82,6 +71,46 @@ func bytesToBits(b []byte, size uint) []bool {
 			bits = append(bits, bit)
 		}
 	}
+	bv.bits = bits
+}
+
+// MarshalSCALE fulfils the SCALE interface for encoding
+func (bv BitVec) MarshalSCALE() ([]byte, error) {
+	buf := bytes.NewBuffer(nil)
+	encoder := scale.NewEncoder(buf)
+	if len(bv.bits) > 268435455 {
```

Comment:
would suggest to keep this value in a constant like `const ARCH32BIT_BITSLICE_MAX_BITS = 268435455` and then use the constant here.

Reply:
I can't find where I read this but I believe this is treated as a constant by the go compiler since it's understood that this inline value will never change.  I'll bring back the `const` anyways since I think it's easier to read.